GVFS/GVFS.FunctionalTests/Tests/FastFetchTests.cs

```diff
@@ -476,16 +480,21 @@ private ProcessResult RunFastFetch(string args)
         {
             args = args + " --verbose";
 
-            string fastfetch = Path.Combine(Settings.Default.CurrentDirectory, "fastfetch.exe");
-            if (!File.Exists(fastfetch))
-            {
-                fastfetch = "fastfetch.exe";
+            string fastfetch;
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                fastfetch = Path.Combine(Settings.Default.CurrentDirectory, "netcoreapp2.1", "fastfetch.dll");
+                File.Exists(fastfetch).ShouldBeTrue();

```

Comment:
Please move this line:

`File.Exists(fastfetch).ShouldBeTrue();`

After the else block.  It should be true on both platforms.